packages/nimble-components/src/rich-text/models/markdown-parser.ts

```diff
@@ -11,6 +11,7 @@ import type { MarkdownParserMentionConfiguration } from './markdown-parser-menti
  * Provides markdown parser for rich text components
  */
 export class RichTextMarkdownParser {
+    private static mentionedHrefs: string[];
```

Comment:
We should have test coverage for the changes to this file in the dedicated test spec for this file. (Note that the specific tests might change depending on what API we decide on for getting the mentioned hrefs; see my comment elsewhere).